MSProject/ViewControllers/DetailBook/WBDetailBookViewController.swift

```diff
@@ -117,18 +154,18 @@ extension WBDetailBookViewController: UITableViewDelegate {
     }
 }
 
-// MARK: - DetailBookDelegate
-extension WBDetailBookViewController: DetailBookDelegate {
+// MARK: - Alert
+extension WBDetailBookViewController {
     func addToWishlist() {
         
     }
     
-    func rentBook() {
-        guard bookViewModel.bookStatus == .available else {
-            showAlertMessage(message: "RENT_NOT_AVAILABLE".localized(withArguments: bookViewModel.bookStatus.bookStatusText()))
-            return
-        }
-        MBProgressHUD.showAdded(to: _view, animated: true)
-        bookDetailViewModel.rentBook(book: bookViewModel)
-    }
+//    func rentBook() {
```

Comment:
🤔

Reply:
😅

Reply:
I wanted to keep this code, because the alert of book unavailable is missing, just a remember to implement it later